component/file_cache/lfu_policy.go

```diff
@@ -158,7 +158,8 @@ func (l *lfuPolicy) clearItemFromCache(path string) {
 		// File was deleted so try clearing its parent directory
 		dirPath := filepath.Dir(path)
 		if dirPath != l.tmpPath {
-			os.Remove(dirPath)
```

Comment:
Lets remove this entire condition/ You can just add the comment saying we need to cleanup empty directories at some point in time.

Reply:
Done